utils/push_to_hub.py

```diff
@@ -0,0 +1,341 @@
+        _generate_replay(model, eval_env, video_length, is_deterministic, repo_local_path)
+
+    # Step 5: Generate the model card
+    generated_model_card, metadata = generate_model_card(
+        algo_name,
+        model_architecture,
+        organization,
+        env_id,
+        mean_reward,
+        std_reward,
+    )
+
+    _save_model_card(repo_local_path, generated_model_card, metadata)
+
+    msg.info(f"Pushing repo {repo_name} to the Hugging Face Hub")
+    repo.push_to_hub(commit_message=commit_message)
+
+    msg.info(f"Your model is pushed to the hub. You can view your model here: {repo_url}")
+    return repo_url
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("--env", help="environment ID", type=str, required=True)
+    parser.add_argument("-f", "--folder", help="Log folder", type=str, required=True)
+    parser.add_argument("--algo", help="RL Algorithm", type=str, required=True, choices=list(ALGOS.keys()))
+    parser.add_argument("-n", "--n-timesteps", help="number of timesteps", default=1000, type=int)
+    parser.add_argument("--num-threads", help="Number of threads for PyTorch (-1 to use default)", default=-1, type=int)
+    parser.add_argument("--n-envs", help="number of environments", default=1, type=int)
+    parser.add_argument("--exp-id", help="Experiment ID (default: 0: latest, -1: no exp folder)", default=0, type=int)
+    parser.add_argument("--verbose", help="Verbose mode (0: no output, 1: INFO)", default=1, type=int)
+    parser.add_argument(
+        "--no-render", action="store_true", default=False, help="Do not render the environment (useful for tests)"
+    )
+    parser.add_argument("--deterministic", action="store_true", default=False, help="Use deterministic actions")
+    parser.add_argument("--device", help="PyTorch device to be use (ex: cpu, cuda...)", default="auto", type=str)
+    parser.add_argument(
+        "--load-best", action="store_true", default=False, help="Load best model instead of last model if available"
+    )
+    parser.add_argument(
+        "--load-checkpoint",
+        type=int,
+        help="Load checkpoint instead of last model if available, "
+        "you must pass the number of timesteps corresponding to it",
+    )
+    parser.add_argument(
+        "--load-last-checkpoint",
+        action="store_true",
+        default=False,
+        help="Load last checkpoint instead of last model if available",
+    )
+    parser.add_argument("--stochastic", action="store_true", default=False, help="Use stochastic actions")
+    parser.add_argument("--seed", help="Random generator seed", type=int, default=0)
+    parser.add_argument(
+        "--env-kwargs", type=str, nargs="+", action=StoreDict, help="Optional keyword argument to pass to the env constructor"
+    )
+    parser.add_argument("-orga", "--organization", help="Huggingface hub organization", default="sb3")
+    # TODO: allow to overwrite repo-id
```

Comment:
As you mentioned we need to have an argument otherwise we can't create a repo.
--repo_id

For instance if I want to create this repo (or update this repo):
ThomasSimonini/ppo-TestMountainCar-v0

`python -m utils.push_to_hub --algo ppo --env CartPole-v1 -f logs/ --repo-id "ThomasSimonini/ppo-TestMountainCar-v0"`